include/fmt/core.h

```diff
@@ -938,9 +938,9 @@ struct arg_data<T, Char, NUM_ARGS, 0> {
   T args_[NUM_ARGS != 0 ? NUM_ARGS : +1];
 
   template <typename... U>
-  FMT_INLINE arg_data(const U&... init) : args_{init...} {}
-  FMT_INLINE const T* args() const { return args_; }
-  FMT_INLINE std::nullptr_t named_args() { return nullptr; }
+  FMT_CONSTEXPR arg_data(const U&... init) : args_{init...} {}
+  FMT_CONSTEXPR const T* args() const { return args_; }
+  FMT_CONSTEXPR std::nullptr_t named_args() { return nullptr; }
```

Comment:
Replacing `FMT_CONSTEXPR` with `constexpr` here (as here https://github.com/alexezeder/fmt/commit/3c9c7c798c789d029876383b459c0206859ab18c) leads to these failed actions:

- Linux - https://github.com/alexezeder/fmt/actions/runs/422137428
- MacOS - https://github.com/alexezeder/fmt/actions/runs/422137426

TLDR:
```
error: 'constexpr' non-static member function will not be implicitly 'const' in C++14; add 'const' to avoid a change in behavior [-Werror,-Wconstexpr-not-const]
```

Reply:
OK, not a big deal.